docs/userstories/v1/egress-control-in-anp/egress-control-outside-cluster.md

```diff
@@ -0,0 +1,56 @@
+
+### Summary
+
+1. **As a** cluster administrator **I want** to restrict traffic from
+specific cluster workloads to all or specific destinations outside the
+cluster **so that** I can enforce security for south-north traffic.
+
+2. **As a** cluster administrator **I want** to explicitly allow traffic
+from cluster workloads to handpicked destinations outside the cluster and
+deny the rest **so that** I can guarantee cluster wide egress traffic
+guardrails irrespective of whatever network policies are defined by the
+developer for specific namespaces.
+
+### Description
+
+Often times cluster administrators want to explicitly control the
+egress traffic from workloads towards external destinations like allowing
+some (not all) workloads to access a secure service outside the cluster.
+
+They might also want to setup cluster wide safeguard rules for egress in
+general so that they don't have to worry about when namespace specific
+policies are created by users at any given point.
+
+### Acceptance Criteria
+
+1. If I add an admin network policy to deny traffic from a specific "subject"
+of workloads towards an external destination (optionally provide port
+restrictions) then those workloads should not be able to connect to that
+external destination.
+
+2. If I apply an admin network policy with lower priority egress rules to deny
+traffic from a specific "subject" of workloads towards all external
+destinations and then apply higher priority egress rules to allow traffic for
+those workloads towards well defined destinations then those workloads should
+not be able to connect to any other external destinations than the explicitly
+allowed ones.
+
```

Comment:
I think you need another criterion for the "guardrails" case?

Reply:
I thought point 2 was the criterion for the "guardrails" case.. :D I changed the word guardrail, ptal the second iteration to see if it makes sense.